packages/synapse-react-client/test/containers/entity/annotations/SchemaDrivenAnnotationsEditor.integration.test.tsx

```diff
@@ -105,19 +129,42 @@ describe.skip('SchemaDrivenAnnotationEditor tests', () => {
     },
   )
 
-  const annotationsHandler = rest.get(
+  // Returns an entity with annotations that match the schema
+  const annotationsWithSchemaHandler = rest.get(
```

Comment:
Following [PLFM-6811](https://sagebionetworks.jira.com/browse/PLFM-6811), the annotation returned by Synapse may change depending on the presence of a schema.

For example, all "custom annotations" are arrays, but if a single annotation is described by a bound schema as a string, Synapse will return a string.

For this reason, this handler needed to be split into two different handlers.